packages/flutter_tools/test/general.shard/asset_bundle_variant_test.dart

```diff
@@ -3,90 +3,143 @@
 // found in the LICENSE file.
 
 import 'dart:convert';
+import 'dart:io' as io;
```

Comment:
What's so illegal about it? ☹️

Reply:
Generally we don't want code to use dart:io directly because it is hard to test. Making tests depend on their real host platforms (if its not an integration test) leads to hard to debug issues if you don't have that machine in question. Anecdotally, folks are more likely to skip a test rather than investigate if its failing only on a single platform